roles/kubernetes/node/defaults/main.yml

```diff
@@ -12,17 +12,21 @@ kube_resolv_conf: "/etc/resolv.conf"
 kubelet_enforce_node_allocatable: "\"\""
 
 # Set runtime and kubelet cgroups when using systemd as cgroup driver (default)
-kubelet_runtime_cgroups: "/systemd/system.slice"
-kubelet_kubelet_cgroups: "/systemd/system.slice"
+kubelet_runtime_cgroups: "{{ kube_reserved_cgroups }}/{{ container_manager }}.service"
+kubelet_kubelet_cgroups: "{{ kube_reserved_cgroups }}/kubelet.service"
```

Comment:
After the PR , the default kubelet_kubelet_cgroups, would change from the /systemd/system.slice to the /kube.slice/kubelet.service. And not managed by the systemd.

So is the default kubelet config changed after the PR?

Reply:
I'm sorry, I misunderstood the yellow "Pending" decorator on my comments, so the above comment was just sent (it was actually commented many days ago).